auth/auth_test.go

```diff
@@ -27,59 +28,64 @@ type User struct {
 
 type AuthTestSuite struct {
 	suite.Suite
-	auth *Auth
+	auth       *Auth
+	mockCache  *cachemock.Cache
+	mockConfig *configmock.Config
+	mockOrm    *ormmock.Orm
+	mockDB     *ormmock.Query
 }
 
 func TestAuthTestSuite(t *testing.T) {
 	unit = time.Second
-	suite.Run(t, &AuthTestSuite{
-		auth: NewAuth(guard),
-	})
+
+	suite.Run(t, new(AuthTestSuite))
 }
 
 func (s *AuthTestSuite) SetupTest() {
-
+	s.mockCache = &cachemock.Cache{}
+	s.mockConfig = &configmock.Config{}
+	s.mockOrm = &ormmock.Orm{}
+	s.mockDB = &ormmock.Query{}
+	s.auth = NewAuth(guard, s.mockCache, s.mockConfig, s.mockOrm)
 }
 
 func (s *AuthTestSuite) TestLoginUsingID_EmptySecret() {
-	mockConfig := mock.Config()
```

Comment:
All of the test cases shouldn't be dependent on the `github.com/goravel/framework/testing/mock` package, will cause import cycle error, use `&cachemock.Cache{}` instead.